common/libs/signing.ts

```diff
@@ -26,16 +33,23 @@ export function signMessageWithPrivKey(
   const fullMessage = msg + spacer + date;
   const hash = sha3(fullMessage);
   const signed = ecsign(hash, privKey);
-  const combined = Buffer.concat([
```

Comment:
Using https://github.com/MetaMask/eth-simple-keyring/blob/master/index.js#L61 as a reference, they also concentenate `r,s,v` and return the hex string and `0x` prefixed result. Why did this need to be changed?

Reply:
Easier to handle as an object then having to reparse it from a hex string. Seemed redundant to do so.